content/project/governance.adoc

```diff
@@ -4,233 +4,227 @@ title:  Project Governance Document
+The name "Jenkins" is a registered trademark in the USA (link:https://trademarks.justia.com/854/47/jenkins-85447465.html[#4664929], held by SPI) in order to protect the project and users from confusing use of the term. To reduce the process overhead and uphold our open-source spirit, we adopt the Linux kernel policy on trademark usage.
 
-You need to apply for a [sublicense|Trademark Sublicense] if you are using the term "Jenkins" as part of your own trademark or brand identifier for Jenkins-based software goods or services. It doesn’t matter if your trademark is unregistered, or if you do not plan to make any money using the mark.
+You need to apply for a link:https://wiki.jenkins-ci.org/display/JENKINS/Trademark+Sublicense[sublicense] if you are using the term "Jenkins" as part of your own trademark or brand identifier for Jenkins-based software goods or services. It doesn’t matter if your trademark is unregistered, or if you do not plan to make any money using the mark.
 
 Answering the following questions (which break out each of the key issues) may help you determine if you need a sublicense. If you are still in doubt, please contact the board and we will work with you to determine whether you need to apply for a sublicense.
 
 If the answer to all three of the following questions is "yes," then you need to apply for a sublicense. If the answer to any of these questions is "no," then you do not need to apply for a sublicense.
 
-# Is my mark a trademark (see how Linux Foundation define "trademark" in [FAQ|http://www.linuxfoundation.org/programs/legal/trademark/faq])?
-# Does my mark contain the following string of adjacent letters, in this order: "Jenkins"? These letters may or may not be capitalized, and in the case of foreign characters, phonetic translations also apply.
-# Do I use my mark to identify software-related goods or services (see how that phrase is [defined|http://www.linuxfoundation.org/programs/legal/trademark/faq] again in LF)?
+- Is my mark a trademark (see how Linux Foundation define "trademark" in link:http://www.linuxfoundation.org/programs/legal/trademark/faq[FAQ])?
+- Does my mark contain the following string of adjacent letters, in this order: "Jenkins"? These letters may or may not be capitalized, and in the case of foreign characters, phonetic translations also apply.
+- Do I use my mark to identify software-related goods or services (see how that phrase is link:http://www.linuxfoundation.org/programs/legal/trademark/faq[defined] again in LF)?
 
-A list of trademark usages approved by the project can be found on the [Approved Trademark Usage] page.
+A list of trademark usages approved by the project can be found on the link:https://wiki.jenkins-ci.org/display/JENKINS/Approved+Trademark+Usage[Approved Trademark Usage] page.
 
-h2. Project roles/stakeholders
+== Project roles/stakeholders
 
-{anchor:board}
```

Comment:
I believe I added that so I could link to it from another page (I'll try and find which one).

So it might be nice to keep the anchor (in .adoc format).